api/config/cdm/cdm_5_2.json

```diff
@@ -934,6 +934,25 @@
           "name": "qualifier_source_value",
           "mode": "nullable",
           "description": "The source value associated with a qualifier to characterize the observation"
+        },
+        {
+          "type": "integer",
+          "name": "value_source_concept_id",
+          "mode": "nullable",
+          "description": "A foreign key to a Concept for the value in the source data. This is applicable to observations where the result can be expressed as a non-standard concept.",
```

Comment:
Did you write these? If not where did they come from?

Reply:
Added stuff to the README about these.

FWIW, I'm considering updating the schema we copied these from in curation to include them, at which point these instructions would need to be updated.